.github/workflows/reusable-crn-analytics-algolia-sync.yml

```
@@ -99,7 +99,7 @@ jobs:
           ALGOLIA_API_KEY: ${{ steps.algolia-keys.outputs.algolia-api-key }}
           ALGOLIA_APP_ID: ${{ steps.algolia-keys.outputs.algolia-app-id }}
           ALGOLIA_INDEX_TEMP: '${{ steps.setup.outputs.crn-analytics-algolia-index }}_${{ github.run_id }}_temp'
-          ENTITY_TYPE: ${{ (inputs.metric == 'team-leadership' && 'team-leadership') || (inputs.metric == 'team-productivity' && 'team-productivity') || (inputs.metric == 'user-productivity' && 'user-productivity')}}
+          ENTITY_TYPE: ${{ (inputs.metric == 'team-leadership' && 'team-leadership') || inputs.metric == 'engagement' || (inputs.metric == 'team-productivity' && 'team-productivity') || (inputs.metric == 'user-productivity' && 'user-productivity')}}
```

I believe we need to add `&& 'engagement'` because this should give us the entity type not a boolean :)
```suggestion
          ENTITY_TYPE: ${{ (inputs.metric == 'team-leadership' && 'team-leadership') || (inputs.metric == 'engagement' && 'engagement') || (inputs.metric == 'team-productivity' && 'team-productivity') || (inputs.metric == 'user-productivity' && 'user-productivity')}}
```